### DISCUSSION TITLE
Server Error in '/CMAppCreator' Application

### DISCUSSION THREAD

**Discussion by jwellnitz-banderaelectriccom** [CBennerstedt/CMAppCreator#4] [Q&A] [answered]
Hello!
I love the concept of this application. I would love to be able to configure it and deploy in our environment, but we are having an issue with a Parser Error.

I have followed the instructions you have provided in the README, and unless I'm overlooking something there appears to be a problem for our deployment.  I am not well versed in ASPX or .NET so I'm hoping you may be able to provide some guidance.

![Capture](https://user-images.githubusercontent.com/95879324/145457565-4aa26a15-b25b-4a4f-8f6b-3aa93e0ef358.PNG)

**Comment by CBennerstedt** [comment, marked as answer]
Hello!

Glad you like the idea!
Very sorry for such a late response (in my defense, my second child was born and I've been on paternity leave and pretty much offline last few weeks)

The error suggests the application is missing one of the required DLLs (AdminUI.DcmObjectWrapper.dll) that you need to provide from your own Config Manager environment where you need to copy necessary DLLs from an existing installation of your Config Manager Console application to the IIS folder and the bin folder of the Webapp.

To fix this, please follow the  the installation instructions under section 1-3 and you should be good to go.

Regards Christoffer
